content/patterns/multicloud-gitops/getting-started.md

```diff
@@ -80,10 +75,10 @@ service](https://console.redhat.com/openshift/create).
     export KUBECONFIG=~/my-ocp-env/hub/auth/kubconfig
     ```
 
-3. Apply the changes to your cluster.
+2. Deploy the pattern to your cluster.
 
     ```sh
-    make install
+    ./pattern.sh make install
     ```
 
 4. Verify that the Operators have been installed.
```

Comment:
Out of scope for this PR but noting it here for tracking we will need to convert this step to a heading (Deploying the cluster using the Validated Patterns Operator).